pkg/nsx/services/vpc/vpc.go

```diff
@@ -781,3 +808,23 @@ func (vpcService *VPCService) getLBProvider() string {
 	}
 	return LBProviderAVI
 }
+
+func (service *VPCService) GetVPCFromNSXByPath(vpcPath string) (*model.Vpc, error) {
+	vpcResInfo, err := common.ParseVPCResourcePath(vpcPath)
+	if err != nil {
+		log.Error(err, "failed to parse VPCResourceInfo from the given VPC path %s", vpcPath)
```

Comment:
It seems that the logger doesn't support the way likes printf.

Reply:
updated.